lib/iterable_flutter.dart

```diff
@@ -36,7 +36,13 @@ class IterableFlutter {
   }
 
   static Future<void> track(String event) async {
-    await _channel.invokeMethod('track', event);
+    await _channel.invokeMethod('track');
```

Comment:
Any reason for deleting the event here? the track will be broken without it

Reply:
Oh shoot, this is my mistake. I didn't mean to remove it